docs/getting-started.md

```diff
@@ -0,0 +1,158 @@
+- **Configuration**: All necessary scripts are located in the
+`stack/keycloak/migration` folder.
+Extensions to Keycloak should be implemented through these scripts.
+
+#### PostgreSQL
+
+- **Website**: <https://www.postgresql.org>
+- **Purpose**: Run a PostgreSQL database instance locally
+
+#### pgAdmin
+
+- **Website**: <https://www.pgadmin.org/>
+- **Purpose**: Run a database management UI pre-configured to connect to the local PostgreSQL instance
+
+#### API gateway
+
+- **Website**: <https://refarch.oss.muenchen.de/gateway>
+- **Purpose**: Launch the API gateway of the RefArch in your local environment
+- **Configuration**: The API gateway behaviour can be adjusted by altering the respective environment variables
+defined in `stack/docker-compose.yml` under the key `refarch-gateway.environment`
+
+#### Appswitcher Server
+
+- **Repository**: <https://github.com/it-at-m/appswitcher-server>
+- **Purpose**: Server component required for appswitcher-vue to access local development tools via the frontend UI
+- **Configuration**: Configure the AppSwitcher within the `stack/appswitcher-server`
+folder.
+
+## Backend & EAI
+
+### Prerequisites
+
+- Java 21
+- Maven 3.9.7
+- Docker (for Postgres and Keycloak)
+
+### Configure templates
+
+After getting the templates, you will need to make a few adjustments:
+
+1. Rename the folder you copied (e.g. from `refarch-backend` to `myapp-backend`).
+2. Change the package name from `de.muenchen.refarch` to
+   `de.muenchen.YourPackageName` (It's recommended to use an IDE for this
+   task, as it will automatically update the imports; otherwise, you will
+   need to do this manually).
+3. Inside the `pom.xml`, update the `artifactId`, `groupId`, and `name`
+   fields.
+4. Install required dependencies by executing the command `mvn install`.
+
+Congratulations, the Backend/EAI is now ready to use.
+
+### Profiles
+
+By default, the template supports two Maven profiles:
+
+- `local`: Contains the necessary configurations for running the application
+  locally, including settings for ports, SSO, and the database
+- `no-security`: Configured to operate without security measures for development
+  or testing purposes.
+
+## Frontend & Web Components
+
+### Prerequisites
+
+- Node.js version 20 to 22
+- Docker (for AppSwitcher)
+
+### Configure Templates
+
+After getting the templates, make the following adjustments:
+
+1. Rename the folder you copied (e.g. from `refarch-frontend` to `myapp-frontend`).
+2. Change the artifact `name` inside the `package.json` file.
+3. Generate a new `package-lock.json` file by executing the command `npm install`.
+
+Congratulations, the Frontend/WebComponent is now ready to use.
+
+## Next steps
+
+If you finished the template configurations for all required components your project needs
+you have now successfully set up your project.
+
+To help you with software developing and project organisation, please check out **TODO** and **TODO**.
```

Comment:
Why don't you add dead links as i did? Think that's more useful than that or just write where it should link in the feature

Reply:
Actually we don't know how many links there will be yet so I just set it to TODO. 😄